woocommerce/woocommerce-android
# Issue/5924 pending order payment link

Closes #5924 - adds the ability to share the payment link in order details for pending orders.

To test:

* View order detail for a pending order
* Note that the options menu is visible with a "Share payment link" menu item
* Verify share sheet appears when that menu item is tapped

* View order detail for an order that  has a status other than pending
* Verify the options menu doesn't appear


* View an order with pending status
* Change the status
* Verify the options menu disappears

* View an order without pending status
* Change the status to pending
* Verify the options menu appears


https://user-images.githubusercontent.com/3903757/164475380-aa29db06-b88f-4b99-9962-3c8c0d51865e.mp4
